api/kogitoinfra_types.go

```diff
@@ -24,10 +24,10 @@ import (
 type KogitoInfraConditionType string
 
 const (
-	// SuccessInfraConditionType ...
-	SuccessInfraConditionType KogitoInfraConditionType = "Success"
-	// FailureInfraConditionType ...
-	FailureInfraConditionType KogitoInfraConditionType = "Failure"
+	// KogitoInfraSuccess ...
+	KogitoInfraSuccess KogitoInfraConditionType = "Success"
+	// KogitoInfraFailure ...
+	KogitoInfraFailure KogitoInfraConditionType = "Failure"
```

Comment:
We don't need to replicate these constants here. Please use this instead:
https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/types.go#L719-L722

Reply:
@ricardozanini The link that you had shared above are status but we are using `KogitoInfraSuccess` & `KogitoInfraFailure` as type. In Condition struct we have two fields `type` and `status`. Every type could have its status `success` or `failure`. 
To me as well these type are not looks right, type should be something like `deployed` or `provisioning` which could have their status as `success` or `false`.

Reply:
Agreed. We should change this naming then :/